### ISSUE TITLE
Unnecessary travis builds

### ISSUE THREAD

**Issue by anweshknayak** [openMF/web-app#94]
We don't need all the 4 builds. 1 or 2 would suffice as each build perform the same tasks.

**Comment by abhaychawla** [comment]
I didn't understand which four builds you are referring to. There is one build which is executed for the PRs and one once the PR is merged. Both are required, to ensure that PR can be merged and then the project can be deployed to gh-pages, right?

**Comment by anweshknayak** [comment]
for gh-pages build I think it is done once the PRs are merged.

- Travis CI - Branch 
- continuous-integration/travis-ci/push 
are not necessary.

Only one travis ci build should should suffice

**Comment by abhaychawla** [comment]
These builds occur once the PRs are merged. They are necessary without them it is not possible to deploy the project to gh-pages.

**Comment by anweshknayak** [comment]
Okay will see more into this. IMO two builds are sufficient. One for pr and push. that should work for gh pages

**Comment by abhaychawla** [comment]
Yes, the two builds our different for the PR and push. We require the Github token environment variable in for the project to be built and changes pushed to gh-pages. So what happens in the build for PR is that the token is not read due to security reasons. It is read-only when the PR is merged and Travis build is executed again. So we will require both of them.
